doc/usermanual/slideshow/slideshow.xml

```diff
@@ -66,15 +66,31 @@
           <tbody>
             <row>
               <entry>
-                <emphasis>left-click</emphasis>
+                <emphasis>+</emphasis> or <emphasis>up</emphasis>
+              </entry>
+              <entry>
+                increase delay between each image
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <emphasis>-</emphasis> or <emphasis>down</emphasis>
+              </entry>
+              <entry>
+                decrease delay between each image
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <emphasis>left-click</emphasis> or <emphasis>shift-right</emphasis> or <emphasis>right</emphasis>
```

Comment:
I think I'd go for "left click or right or shift-right" (same for left below): it seems more logical to me to give the "easier" (single press shortcut vs double press) way first.

Reply:
Done.